frontend/src/app/commonComponents/MenuButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  Menu,
+  MenuButton as ReactMenuButton,
+  MenuItem,
+} from "@szhsin/react-menu";
+import "@szhsin/react-menu/dist/index.css";
+import "./MenuButton.scss";
+
+interface Props {
+  title?: string;
+  id?: string;
+  items: {
+    name: string;
+    content?: React.ReactElement;
+    action: () => void;
+  }[];
+  buttonContent: React.ReactElement;
```

Comment:
Why is this defined as an element instead of a node?

Reply:
This can be interchanged with a node, and has been done so in the latest commit :) Thank you for catching this